### ISSUE TITLE
UITK - Support React 17

### ISSUE THREAD

**Issue by joshwooding** [jpmorganchase/salt-ds#1301]

**Comment by origami-z** [comment]
Remaining unit test failures
- FAIL packages/tree/src/__tests__/Tree.accessibility.test.tsx
- FAIL packages/data-grid/src/__tests__/AccessibleDataGrid.test.js
- FAIL packages/form-renderer/src/__tests__/FormRenderer.test.tsx
- FAIL packages/comments/src/__tests__/Comments.test.js
- FAIL packages/forms/src/__tests__/Form.test.tsx

**Comment by joshwooding** [comment]
some cypress tests still failing, ETA end of next wednesday
